src/test/starknet.t.sol

```diff
@@ -145,4 +157,118 @@ contract StarknetTests is GoerliDssSpellTestBase, ConfigStarknet {
 
         assertTrue(core.isNotFinalized());
     }
+
+    function testTeleportFW() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        address router = addr.addr("MCD_ROUTER_TELEPORT_FW_A");
+        StarknetTeleportBridgeLike bridge = StarknetTeleportBridgeLike(addr.addr("STARKNET_TELEPORT_BRIDGE"));
+        address escrow = addr.addr("STARKNET_ESCROW");
+
+        bytes32 ilk = "TELEPORT-FW-A";
+        bytes23 domain = "ETH-GOER-A";
+
+        emit log_address(address(spell));
+        emit log_address(addr.addr("STARKNET_TELEPORT_BRIDGE"));
+        emit log_address(addr.addr("STARKNET_TELEPORT_FEE"));
+
+        assertEq(bridge.escrow(), escrow);
+        assertEq(bridge.teleportRouter(), address(router));
+        assertEq(bridge.dai(), address(dai));
+
+        checkTeleportFWIntegrationInternals(
+            "STA-GOER-A",
+            domain,
+            100_000 * WAD,
+            address(bridge),
+            addr.addr("STARKNET_TELEPORT_FEE"),
+            escrow,
+            100 * WAD,
+            WAD / 10000,   // 1bps
+            8 days
+        );
+        assertEq(bridge.l2DaiTeleportGateway(), starknetValues.l2_teleport_gateway);
+
+    }
+
+    function testCureTeleport() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        bytes23 domain = "ETH-GOER-A";
+
+        checkCureLoadTeleport(
+            "STA-GOER-A",
+            domain,
+            100_000 * WAD,
+            TeleportFeeLike(addr.addr("STARKNET_TELEPORT_FEE")).fee(),
+            100_000 * RAD,
+            true
+        );
+    }
+
+    function checkTeleportFWIntegrationInternals(
```

Comment:
this should go in base spell tests in `Goerli-DssSpell.t.base.sol` (see previous patterns, `PE-1045`)